include/display.h

```diff
@@ -0,0 +1,69 @@
+//
+// Project name : LAB 05 - Refactor
+// Project members :
+// - Florian Duruz
+// - Aellen Quentin
+//
+
+#ifndef REFACTOR_DISPLAY_H
+#define REFACTOR_DISPLAY_H
```

Comment:
Change `REFACTOR_DISPLAY_H` to something like `DISPLAY_HEADER`

Reply:
Done